maintainers/maintainer-list.nix

```
@@ -2151,6 +2151,12 @@
     githubId = 18467667;
     name = "Alexander Bantyev";
   };
+  bananad3v = {
+    email = "banana@banana.is-cool.dev";
+    github = "bananad3v";
```

```suggestion
    github = "BANanaD3V";
```
https://github.com/NixOS/nixpkgs/blob/5164ebdf667c299c4e688918b87d8606c0b49707/maintainers/maintainer-list.nix#L25